skysurvey/target/afterglow.py

```diff
@@ -1,19 +1,11 @@
-
 import numpy as np
 from scipy import stats
-
 import sncosmo
-try:
-    import afterglowpy
-except:
-    raise ImportError("could not import afterglowpy ; run pip install afterglowpy")
-
 
-from .core import Transient
-
-
-__all__ = ["Afterglow"]
+import afterglowpy
```

Comment:
You should leave:

try:
    import afterglowpy
except:
    raise ImportError("could not import afterglowpy ; run pip install afterglowpy")

as before.